docs/binary_input.md

```diff
@@ -19,11 +19,15 @@ is out of range it will be adjusted to the nearer border. For example, when mode
 - if input shape is [1,90,200,3] it will be resized into [1,100,200,3]
 - if input shape is [1,220,200,3] it will be resized into [1,200,200,3]
 
-Processing the binary image requests requires the model or the custom nodes to accept NHWC layout in BGR color 
+When the endpoint (model or pipeline) shape supports any dimension value (`-1`), the image is not resized and it is enforced to have resolution aligned for entire batch of images.
+
+Resize functionality is also disabled by default for endpoints with layouts different than `...` and `NHWC` (or `N?HWC`, when modified by demultiplexer).
```

Comment:
With such layout, I guess the binary format is not supported at all

Reply:
Binary format is supported for those layouts. NHWC, N?NHWC (in case of demultiplexer) and ... (in case of single custom node at the DAG entry)

Reply:
What do you mean by that?